lib/install/template.rb

```diff
@@ -12,7 +12,7 @@
 copy_file "#{__dir__}/config/.babelrc", ".babelrc"
 
 puts "Creating javascript app source directory"
-directory "#{__dir__}/javascript", "#{Webpacker::Configuration.source}"
+directory "#{__dir__}/javascript", "app/javascript"
```

Comment:
If someone re-runs `webpacker:install` this will re-create `app/javascript`if the source path is not the default we ship